samples/sample_hevc_fei/src/hevc_fei_encode.cpp

```diff
@@ -223,6 +294,12 @@ mfxStatus FEI_Encode::EncodeFrame(mfxFrameSurface1* pSurf)
     for (;;)
     {
 
+        if (m_pFile_repack_ctrl.get() && pSurf)
+        {
```

Comment:
All ctrl parameters are set in SetCtrlParams function. Why is this block here? I see that it's implemented to work only in Encoded Order.

Reply:
For the flushed frame encode request, don't need to do SetRepackCtrl again. Yes, the repack ctrl tests only work in encoded order, for the repack ctrl data is always in encoded order. AVC works in this way as well.